wemake_python_styleguide/visitors/ast/builtins.py

```diff
@@ -198,7 +201,9 @@ def _check_is_magic(self, node: ast.Constant) -> None:
             isinstance(parent, self._allowed_parents)
             or node.value in constants.MAGIC_NUMBERS_WHITELIST
             or is_non_magic
-            or self._check_is_number_in_typing_literal(parent)
+            or check_is_node_in_specific_annotation(
+                parent, 'Literal', {'typing', 'typing_extensions'}
```

Comment:
Please, move `{'typing', 'typing_extensions'}` to class level and make it a `frozenset`